.github/ISSUE_TEMPLATE/config.yml

```
@@ -1 +1,17 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Discussions
+    url:  https://github.com/FreeTubeApp/FreeTube/discussions/categories/general
+    about: View discussions or start one yourself
+  - name: Questions?
```

Oh sorry about this. My brain did not register that i placed that q mark

```suggestion
  - name: Questions
```